packages/financial-templates-lib/src/logger/Transports.ts

```diff
@@ -74,6 +80,13 @@ export function createTransports(transportsConfig: TransportsConfig = {}): Trans
       );
     }
 
+    // If there is a discord ticket config, create a new transport.
+    if (transportsConfig.discordTicketConfig || process.env.DISCORD_TICKET_CONFIG) {
+      const discordTicketConfig =
+        transportsConfig.discordTicketConfig ?? JSON.parse(process.env.DISCORD_TICKET_CONFIG || "null");
+      transports.push(new DiscordTicketTransport({ level: "info" }, discordTicketCreateConfig(discordTicketConfig)));
```

Comment:
If we wanted to have the transport be able to pass some log that would be picked up by PD, maybe we could pass in a callback here that (after initialization) would call logger.error so it would show up in PD? Thoughts?

Reply:
We can modify the AugmentedLogger to attach child transportErrorLogger that is created only from PD transport. After the transports are fixed to return callback(error) on log transport errors we could add logger error event handler to alert on log transport issues through this transportErrorLogger